### DISCUSSION TITLE
Is V programming language based on assembly code?

### DISCUSSION THREAD

**Discussion by yingshaoxo** [vlang/v#17211] [Questions and Answers]
Why I didn't see any assembly code in the project?

**Comment by JalonSolov** [comment]
V is not written in assembler, but it does support inline assembly with `asm` blocks.

https://github.com/vlang/v/blob/master/doc/docs.md#inline-assembly

**Comment by MatejMagat305** [comment]
V is compile into C, JS, GO, there is native backend which IS compile to machine code, but is verry imature (it can compile minimum features) and now is main work on C (JS, GO is second), but as @JalonSolov  say it suport asm block as well as C
